### PR TITLE
Assign asset id feature access block

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala [hunk 1]
```diff
@@ -70,7 +71,7 @@ class FileService(
         val parentId = parentNode.map(_.id)
         val parentFileReference = parentNode.flatMap(_.reference)
         val fileId = treeNode.id
-        val assetId = uuidSource.uuid
+        val assetId = if (blockAssignAssetId) None else Some(uuidSource.uuid)
```
Comment (vimleshtna) [inline src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala:74]: Can you add a test for if `blockAssignAssetId` is `true` then the value should be `None` for `assetId`?

**Review by vimleshtna — COMMENTED** [review]
Looks good. Just a minor comment. Thanks

**Review by vimleshtna — APPROVED** [review]
Looks good. Thanks
